ports/servoshell/egl/app.rs

```diff
@@ -261,7 +261,7 @@ pub struct App {
     platform_window: Rc<EmbeddedPlatformWindow>,
 }
 
-#[allow(unused)]
+#[expect(unused)]
```

Comment:
I think this is used in https://github.com/servo/servo/blob/b974a6685cd1134f7b637076e792632bbe71a097/ports/servoshell/egl/ohos/mod.rs#L241-L254 @mrobinson can you confirm this is used and therefore we can remove the expectation?